security/certificates/api-server.adoc

```diff
@@ -5,9 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The default API server certificate is issued by an internal {product-title}
-cluster CA. Clients outside of the cluster will not be able to verify the
-API server's certificate by default. This certificate can be replaced
-by one that is issued by a CA that clients trust.
+The default API server certificate is issued by an internal {product-title} cluster CA. Clients outside of the cluster cannot verify the API server's certificate by default. You can add a certificate, which is issued by a CA that clients trust, for the domains specified in your certificate to use.
+
+[NOTE]
+====
+The `oc login` commands displayed from the web console during authentication always use the self-signed CA that matches the external self-signed load balancer certificate that is generated and rotated by the Kubernetes API Server Operator. It is currently not possible to override this with a custom domain.
```

Comment:
I am not sure what this means: _The `oc login` commands displayed from the web console during authentication_. Wil this be obvious to a typical user?

Reply:
Good question, I've reworded to be clearer!